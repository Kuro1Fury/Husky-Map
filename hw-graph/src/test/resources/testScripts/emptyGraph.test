# emptyGraph test
# Tests that an empty graph can be created without any nodes and edges

# Create an empty graph
CreateGraph graph1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
