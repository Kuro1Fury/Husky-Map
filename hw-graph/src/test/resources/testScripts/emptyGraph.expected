# emptyGraph test
# Tests that an empty graph can be created without any nodes and edges

# Create an empty graph
created graph graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: